src/server/pbsd_init.c

```diff
@@ -317,6 +318,15 @@ init_server_attrs()
 	attr_jobscript_max_size.at_type  |= ATR_TYPE_SIZE;  /* get_bytes_from_attr() is checking for at_type */
 	set_size(&attr_jobscript_max_size,&attrib,SET);
 
+	sprintf(dflt_log_event, "%d", SVR_LOG_DFLT);
```

Comment:
I can use TOSTR but since SVR_LOG_DFLT is an ORed expression of other macros it will look similar to this in the string "0x01|0x02|0x04|0x08". 

This is why I need a string conversion